a10_neutron_lbaas/plumbing/vthunder_per_tenant.py

```diff
@@ -62,7 +63,8 @@ def _create_instance(self, tenant_id, a10_context, lbaas_obj, db_session):
             'tenant_id': tenant_id,
             'nova_instance_id': instance['nova_instance_id'],
             'name': instance['name'],
-            'host': instance['ip_address']
+            'host': instance['ip_address'],
+            '_perform_initialization': True
```

Comment:
It's not always true. This method is creating the instance. Since it's being created it's new and needs initialization.